frontend/locales/fr.ts

```diff
@@ -1,7 +1,864 @@
+                loginFailed: "L'authentication a échoué",
+                hangarAuth: 'Impossible de se connecter à HangarAuth',
+                loginDisabled: 'La connexion est temporairement indisponible, veuillez réessayer plus tard',
+                fakeUserEnabled: 'Le faux utilisateur est activé. {0} est donc désactivé',
+            },
+        },
+        createNew: 'Créer un(e) nouveau/nouvelle...',
+        new: {
+            project: 'Nouveau projet',
+            organization: 'Nouvelle organisation',
+        },
+        hangar: {
+            home: "Page d'accueil",
+            forums: 'Forums',
+            code: 'Code',
+            docs: 'Documentations',
+            javadocs: 'JavaDocs',
+            hangar: 'Hangar (Plugins)',
+            downloads: 'Téléchargements',
+            community: 'Communauté',
+        },
+    },
+    project: {
+        stargazers: "Observateurs d'étoiles",
+        noStargazers: "Il n'y a pas encore d'observateurs d'étoiles sur ce projet 😢",
+        watchers: 'Observateurs',
+        noWatchers: "Il n'y a pas encore d'observateurs sur projet 😢",
+        members: 'Membres',
+        category: {
+            info: 'Catégorie: {0}',
+            admin_tools: "Outils d'administration",
+            chat: 'Chat',
+            dev_tools: 'Outils de développeur',
+            economy: 'Économie',
+            gameplay: 'Jouabilité',
+            games: 'Jeux',
+            protection: 'Protection',
+            role_playing: 'Jeux de rôles',
+            world_management: 'Gestion du monde',
+            misc: 'Divers',
+        },
+        actions: {
+            unwatch: "Arrêter d'observer",
+            watch: 'Observer',
+            flag: 'Alerter',
+            star: 'Etoiler',
+            unstar: 'Désétoiler',
+            adminActions: "Actions de l'administrateur",
+            flagHistory: 'Historique des alertes ({0})',
+            staffNotes: 'Notes du personnel ({0})',
+            userActionLogs: "Journaux d'actions utilisateur",
+            forum: 'Forum',
+        },
+        flag: {
+            flagProject: 'Alerte {0} ?',
+            flagSend: "Signalé avec succès, merci d'avoir fait de cette communauté un endroit meilleur !",
+            flagSent: 'Alerte soumise pour examen',
+            flags: {
+                inappropriateContent: 'Contenu inapproprié',
+                impersonation: "Usurpation d'identité ou tromperie",
+                spam: 'Pourriel',
+                malIntent: 'Malveillance',
+                other: 'Autre',
+            },
+            error: {
+                alreadyOpen: "Vous n'avez seulement qu'une seule alerte non résolue sur un projet",
+                alreadyResolved: 'Cette alerte est déjà résolue',
+            },
+        },
+        tabs: {
+            docs: 'Documentations',
+            versions: 'Versions',
+            discuss: 'Discussion',
+            settings: 'Paramètres',
+            homepage: "Page d'acceuil",
```

Comment:
```suggestion
            homepage: 'Accueil',
```
*Page* is not necessary.